### PR TITLE
Require admin access to modify ipv4_address (OMS-612)

### PR DESCRIPTION
**Pull request by deemoowoor**

### CODE REVIEW

File: opennode/knot/model/compute.py [hunk 1]
```diff
@@ -181,7 +182,7 @@ class Compute(Container):
 
     permissions(dict(hostname=('read', 'modify'),
                      mac_addr=('read', 'modify'),
-                     ipv4_address=('read', 'modify'),
+                     ipv4_address=('read', 'zope.Security'),
```
Comment (livenson) [inline opennode/knot/model/compute.py:185]: Why was this needed?

Comment (deemoowoor) [inline opennode/knot/model/compute.py:185]: 'modify' is one of the "owner" permissions. The only thing that owner doesn't have is 'zope.Security'. Though with the current implementation it's not needed, indeed.
